aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -3544,8 +3544,7 @@ private void registerConfigSignInProviders(final AWSConfiguration awsConfigurati
                 identityManager.addSignInProvider(GoogleSignInProvider.class);
             }
         } catch (NoClassDefFoundError exception) {
-            Log.w(TAG, "Sign in provider was not registered due to missing optional dependency. " +
-                    "showSignIn() API may not work as expected.", exception);
+            // This is not 90% use-case. Do not log anything.
```

Comment:
Can you update this comment to explain the scenario that causes this exception to be thrown?  You could probably just put something similar to the log message that you are removing.